clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntity.java

```diff
@@ -80,12 +80,15 @@
     void onDisconnection();
   }
 
+  private final AtomicLong sequenceGenerator = new AtomicLong(0L);
+
   private final EntityClientEndpoint<EhcacheEntityMessage, EhcacheEntityResponse> endpoint;
   private final LifeCycleMessageFactory messageFactory;
   private final Map<Class<? extends EhcacheEntityResponse>, List<ResponseListener<? extends EhcacheEntityResponse>>> responseListeners = new ConcurrentHashMap<Class<? extends EhcacheEntityResponse>, List<ResponseListener<? extends EhcacheEntityResponse>>>();
   private final List<DisconnectionListener> disconnectionListeners = new CopyOnWriteArrayList<DisconnectionListener>();
   private final ReconnectDataCodec reconnectDataCodec = new ReconnectDataCodec();
   private volatile boolean connected = true;
+  private volatile UUID clientId = UUID.randomUUID();
```

Comment:
Missed the fact that it needs generated in `configure` context - please do the generation there instead of during class initialisation.